src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/ubuntu/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -46,5 +45,21 @@
     </File>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <ArchitectureSpecificNativeSymbol Include="@(NativeSplittableBinary -> '%(Identity).dbg')"/>
+
+    <AdditionalLibPackageExcludes Include="%2A%2A\%2A.dbg"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.so"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.dll"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\sosdocsunix.txt"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\crossgen"/>
+    <ArchitectureSpecificNativeSymbol Include="..\_.pdb"/>
+
+    <File Include="@(ArchitectureSpecificNativeSymbol)">
+      <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
+      <IsSymbolFile>true</IsSymbolFile>
```

Comment:
No, all of this creates a separate symbol package. Same package id, but different file name (ends in .symbols.nupkg and put in the bin/Product/Linux.x64.Release/.nuget/symbolpkg directory). @dagood can better answer all the reasons why.